app/bundles/EmailBundle/Swiftmailer/Transport/AmazonApiTransport.php

```diff
@@ -0,0 +1,842 @@
+     * @see https://docs.aws.amazon.com/ses/latest/APIReference/API_DeleteTemplate.html
+     */
+    private function deleteSesTemplate($templateName)
+    {
+        $this->logger->debug('Deleting SES template: '.$templateName);
+
+        try {
+            return $this->client->deleteTemplate(['TemplateName' => $templateName]);
+        } catch (AwsException $e) {
+            $this->logger->error('Exception deleting template: '.$templateName.', '.$e->getAwsErrorCode().', '.$e->getAwsErrorMessage());
+            throw new \Exception($e->getMessage());
+        }
+    }
+
+    /**
+     * @param int   $count   number of recipients for us to consume from the ticket bucket
+     * @param array $message
+     *
+     * @return \Aws\Result
+     *
+     * @throws \Exception
+     *
+     * @see https://docs.aws.amazon.com/ses/latest/APIReference/API_SendBulkTemplatedEmail.html
+     */
+    private function sendSesBulkTemplatedEmail($count, $message)
+    {
+        $this->logger->debug('Sending SES template: '.$message['Template'].' to '.$count.' recipients');
+
+        // wait for a throttle token
+        $this->sendTemplateBucketConsumer->consume($count);
+
+        try {
+            return $this->client->sendBulkTemplatedEmail($message);
+        } catch (AwsException $e) {
+            $this->logger->error('Exception sending email template: '.$e->getAwsErrorCode().', '.$e->getAwsErrorMessage());
+            throw new \Exception($e->getMessage());
+        }
+    }
+
+    /**
+     * Parse message into a template and recipients with their respective replacement tokens.
+     *
+     * @param \Swift_Mime_Message $message
+     *
+     * @return array of a template and a message
+     */
+    private function constructSesTemplateAndMessage(\Swift_Mime_Message $message)
+    {
+        $this->message = $message;
+        $metadata      = $this->getMetadata();
+        $messageArray  = [];
+
+        if (!empty($metadata)) {
+            $metadataSet  = reset($metadata);
+            $emailId      = $metadataSet['emailId'];
+            $tokens       = (!empty($metadataSet['tokens'])) ? $metadataSet['tokens'] : [];
+            $mauticTokens = array_keys($tokens);
+            $tokenReplace = $amazonTokens = [];
+            foreach ($tokens as $search => $token) {
+                $tokenKey              = preg_replace('/[^\da-z]/i', '_', trim($search, '{}'));
+                $tokenReplace[$search] = '{{'.$tokenKey.'}}';
+                $amazonTokens[$search] = $tokenKey;
+            }
+            $messageArray = $this->messageToArray($mauticTokens, $tokenReplace, true);
+        }
+
+        $CcAddresses = [];
+        if (count($messageArray['recipients']['cc']) > 0) {
+            $CcAddresses = array_keys($messageArray['recipients']['cc']);
+        }
+
+        $BccAddresses = [];
+        if (count($messageArray['recipients']['cc']) > 0) {
+            $BccAddresses = array_keys($messageArray['recipients']['bcc']);
+        }
+
```

Comment:
$replyToAddresses = [];
        if (isset($messageArray['replyTo']['email'])) {
            $replyToAddresses = [$messageArray['replyTo']['email']];
        }

Reply:
This allow the use of Reply-To